vocabularies/geo.ttl

```diff
@@ -261,9 +265,10 @@ geo:ehCovers
 
 geo:ehDisjoint
 	a rdf:Property, owl:ObjectProperty ;
-	owl:equivalentProperty :sfDisjoint ;
-	rdfs:domain geo:SpatialObject ;
-	rdfs:range geo:SpatialObject ;
+	owl:equivalentProperty :sfDisjoint, :rcc8dc ;
+	rdfs:domain :SpatialObject ;
+	rdfs:range :SpatialObject ;
+	owl:complementOf :sfIntersects ;
```

Comment:
Isn't owl:complementOf more of a class axiom? Perhaps owl:propertyDisjointWith would be better here.